collector/compile-benchmarks/cargo-0.60.0/src/cargo/core/profiles.rs

```diff
@@ -95,6 +95,14 @@ impl Profiles {
                 .dir_names
                 .insert(InternedString::new("doc"), InternedString::new("debug"));
 
+            profile_makers.by_name.insert(
```

Comment:
Why did you need to change the source of the cargo benchmark? It shouldn't be required to change a benchmark to run `clippy` on it.